tests/clusterintegrationtest/_include.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+#
+# Copyright 2025 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+function start_tidb() {
+  cd ../../ || exit 1
+  echo "building tidb-server..."
+  make
+  echo "build successfully"
+
+  cd - || exit 1
+
+  echo "Starting TiUP Playground in the background..."
+  tiup playground nightly --db=1 --kv=1 --tiflash=1 --db.binpath=../../bin/tidb-server &
+
+  sleep 20
+}
+
+function start_tidb_latest() {
+  echo "Starting TiUP Playground in the background..."
+  tiup playground nightly --db=1 --kv=1 --tiflash=1 &
+
+  sleep 20
+}
+
+function build_mysql_tester() {
+  echo "+ Installing mysql-tester"
+  go install github.com/pingcap/mysql-tester/src@314107b26aa8fce86beb0dd48e75827fb269b365
+}
```

Comment:
It is recommended to rename the binary to mysql-tester after executing go install in this way.
```shell
GOBIN=$PWD go install github.com/pingcap/mysql-tester/src@314107b26aa8fce86beb0dd48e75827fb269b365
mv src mysql-tester
```